spec/spec_helper.rb

```diff
@@ -8,7 +8,7 @@
   coverage_criterion :branch
   # primary_coverage :branch
   minimum_coverage 100
-  add_group 'Views', 'app/views'
+  add_filter 'app/views'
```

Comment:
To fix readout on coverage output on rspec

Reply:
Is this flagging that the view isn't covered? If so should we not add tests rather than a filter?

Reply:
Testing ERB with simple coverage is very new and has some issues. We can re-enable once that has been fixed.